app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/AddProductNutritionFactsFragment.java

```diff
@@ -67,6 +70,7 @@
 import openfoodfacts.github.scrachx.openfood.utils.FileUtils;
 import openfoodfacts.github.scrachx.openfood.utils.ProductUtils;
 import openfoodfacts.github.scrachx.openfood.utils.QuantityParserUtil;
+import openfoodfacts.github.scrachx.openfood.utils.StringComparator;
```

Comment:
StringComparator imported but never used?

Reply:
Changed.

OK, so you closed the previous one.
I reported back the changes here.